iOS Client/Reducers/MBAppReducer.swift

```diff
@@ -84,6 +84,11 @@ func podcastsReducer(action: Action, state: PodcastsState?) -> PodcastsState {
         nextState.podcasts = action.podcasts
     case let action as SelectedPodcast:
         nextState.selectedPodcast = action.podcast
+        nextState.player = .playing
+    case let action as ResumePodcast:
+        nextState.player = .playing
+    case let action as PausePodcast:
+        nextState.player = .paused
     default:
```

Comment:
I'm envisioning 
```
case let action as PlayPausePodcast:
    nextState.player = (nextState.player == .playing ? .paused : .playing )
```

Reply:
was having some trouble with the compiler with the ternary operator so just went with an if else block